README.md

```diff
@@ -2,7 +2,7 @@
 
 F# implementation of Facebook [GraphQL query language specification](https://facebook.github.io/graphql).
 
-[![Build and Test](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/dotnetcore.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/dotnetcore.yml)
+[![Build and Test](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml)
 
 [![Join the chat at https://gitter.im/FSharp-Data-GraphQL/community](https://badges.gitter.im/FSharp-Data-GraphQL/community.svg)](https://gitter.im/FSharp-Data-GraphQL/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Probably we should remove this as no one looks there. What do you think?

Reply:
Good point, but I wouldn't do it right now. This virtual space seems to still exist, although it seems not to be used anymore.
For example, we could announce in that virtual space that we'll close it in a few days. Then close it and then remove this reference here.

Reply:
OK, let's leave then